### DISCUSSION TITLE
How to change gitsigns diffview color scheme.

### DISCUSSION THREAD

**Discussion by dil-mnasir** [LazyVim/LazyVim#2463] [Q&A] [answered]
![image](https://github.com/LazyVim/LazyVim/assets/93813665/08a78d23-1999-4738-9dd9-d44d9f523622)
I am using tokyonight color scheme, and what I dont like is , it shows the added hunk  in the 'red' color(left side). I want to change the diffview to look like the github.

**Comment by dpetka2001** [comment, marked as answer]
[Here](https://github.com/lewis6991/gitsigns.nvim/blob/2c2463dbd82eddd7dbab881c3a62cfbfbe3c67ae/doc/gitsigns.txt#L914) are the highlight groups that `gitsigns` uses. And [here](https://github.com/folke/tokyonight.nvim?tab=readme-ov-file#-overriding-colors--highlight-groups) is described how you can overwrite highlight groups in `tokyonight`. Change them to the colors you would like to use. You can take a look at [mine](https://github.com/dpetka2001/dotfiles/blob/020953244348b92e22e16390082a7911d1c680a5/dot_config/nvim/lua/plugins/colorscheme.lua#L27-L62) for how I change some highlight groups. I believe the ones you want to change are the groups [here](https://github.com/folke/tokyonight.nvim/blob/e3301873c1e96903daebb98cc9b5926810bf73dd/lua/tokyonight/theme.lua#L455-L458).

**Reply by dil-mnasir** [reply]
Thank you. It worked. 

Is there any way I can change the slanted white lines for 'GitDelete' to have plain single color

**Reply by dpetka2001** [reply]
No idea. Maybe it would be better to ask about that in the plugin's repo, since they may give you more insight about the plugin.

**Reply by dpetka2001** [reply]
If you manage to find something out about the slanted white lines, please do post here as well about the solution. I would be interested in that. Thanks in advance :smile:

**Reply by dil-mnasir** [reply]
https://github.com/sindrets/diffview.nvim?tab=readme-ov-file#tips-and-faq
    (vimscript): set fillchars+=diff:╱
    (Lua): vim.opt.fillchars:append { diff = "╱" }

**Reply by dpetka2001** [reply]
@mohnasir-cisco thank you for letting me know :D
